### PR TITLE
 default empty json object on json sequence field 

### PR DESCRIPTION
**Pull request by jorgev259**

### CODE REVIEW

**Review by afriestad — APPROVED** [review]
LGTM

File: pages/transcribe.js [hunk 1]
```diff
@@ -203,7 +209,7 @@ const Transcribe = () => {
 
 	const formik = useFormik({
 		initialValues: {
-			sequence: ""
+			sequence: "{}"
```
Comment (vStone) [inline pages/transcribe.js:230]: I tried this as input when trying to flag a bad image and I ran into "TypeError: a.walls is undefined" further down the line. Just a heads up.

Comment (jorgev259) [reply pages/transcribe.js:230]: https://github.com/jorgev259/corridors-of-time/commit/6dbaa57d58200995fb4d242ea62b4ab43cd5653b

**Review by vStone — APPROVED** [review]

**Comment by tims-j** [comment]
This has been fixed and resolved in 
https://github.com/Corridors-of-Time-Transcription/corridors-of-time/commit/a085b971f5f5dae9dcf2330e895b11156ab963c1

**Pull request closed without merging**